src/antsibull/build_ansible_commands.py

```diff
@@ -345,13 +348,32 @@ def _is_alpha(version: PypiVer) -> bool:
     return version.is_prerelease and pre is not None and pre[0] == 'a'
 
 
+def _extract_python_requires(ansible_core_version: PypiVer, deps: t.Dict[str, str]) -> str:
```

Comment:
FWIW I've been extracting this information from the list of Trove classifiers in the past: https://github.com/cherrypy/cherrypy/blob/5964c48/docs/conf.py#L32-L41.

Since `ansible-core` does declare Trove classifiers, that would be the source of truth: https://github.com/ansible/ansible/blob/56c48d1/setup.cfg#L29-L31.

Reply:
This function is mainly here to work around already released versions of Ansible (2.10+) not having `_python` in their `.deps` files and `.build` files. Assuming this PR is merged before we start releasing Ansible 8.0.0a1 (which I hope it is), this function only needs to handle ansible-base 2.10, ansible-core 2.11, 2.12, 2.13, and 2.14. So it doesn't need any more complicated logic than what's already in there :)

Also besides this function has no access to the Trove classifiers, but only to the version number of ansible-base/ansible-core.